tests/conftest.py

```diff
@@ -43,7 +43,11 @@ def _extract_source_code_from_function(function: FunctionType):
 
 
 def _create_module_file(code: str, tmp_path: Path, name: str) -> tuple[str, str]:
-    name = f'{name}_{secrets.token_hex(5)}'
+    # Max path length in Windows is 260. Leaving some buffer here
```

Comment:
Updated this because there was one test case that failed because of an invalid file name

Reply:
Would it be possible to have the change incorporated in a separate PR? :pray:

Reply:
Yes of course. I'll keep it here until the new PR gets merged into main.